src/controllers/user.controller.js

```diff
@@ -1 +1,32 @@
-exports.status = async (req, res, next) => res.json({ message: 'User endpoint is working.' })
\ No newline at end of file
+const User = require('../models/user.model')
+const errorHandler = require('../middlewares/error-handler')
+
+exports.status = async (req, res, next) => res.json({ message: 'User endpoint is working.' })
+
+exports.add_user = async (req, res, next) => {
+  const { title, username, firstname, lastname } = req.body;
+  const newUser = {
+    profile_picture:'update.to.s3.default',
+    title,
+    badges: 'General',
+    firstname,
+    lastname,
+    username,
+    description: '',
+    rank: [
```

Comment:
get rid of this placeholder data. No data should represent that the user has not participated. It will be much harder to sanitize the data when iterating through for analysis. (eg: having a condition to filter out a contest-number of -1, or what about when we want to calculate average ranking, we would need to avoid -1)